packages/vue-language-core/src/utils/directorySharedTypes.ts

```diff
@@ -17,6 +17,7 @@ type __VLS_Prettify<T> = {
 	[K in keyof T]: T[K];
 } & {};
 
+// TODO: revert this once microsoft/TypeScript#54630 is fixed
```

Comment:
Can't you insert a normal link? :)
It's easier to just click or copy&paste then having to look it up.